test/Chainweb/Test/Pact/RemotePactTest.hs

```diff
@@ -212,13 +210,13 @@ localChainDataTest iot nio = do
           pm = Pact.PublicMeta pactCid "sender00" 1000 0.1 (fromInteger ttl)
 
     expectedResult (PObject (ObjectMap m)) = do
-          assert' "block-height" (PLiteral (LInteger 0))
-          let u = UTCTime (view (from gregorian) (YearMonthDay 1970 1 1)) (fromSeconds (0 :: Int))
-          assert' "block-time" (PLiteral (LTime $ (view (from utcTime)) u))
+          -- assert' "block-height" (PLiteral (LInteger 0))
+          -- let u = UTCTime (view (from gregorian) (YearMonthDay 1970 1 1)) (fromSeconds (0 :: Int))
+          -- assert' "block-time" (PLiteral (LTime $ (view (from utcTime)) u))
           assert' "chain-id" (PLiteral (LString "8"))
           assert' "gas-limit" (PLiteral (LInteger 1000))
           assert' "gas-price" (PLiteral (LDecimal 0.1))
-          assert' "prev-block-hash" (PLiteral (LString ""))
+          -- assert' "prev-block-hash" (PLiteral (LString ""))
```

Comment:
@emmanueldenloye These are the tests you were addressing, right? None of these will work as is, so it might be in our best interest to remove them in favor of your new tests

Reply:
Just remove the commented tests for now as you've verified they're nondeterministic.